rules/rules.json

```diff
@@ -21,8 +21,8 @@
             "replace": "youtube.fuzzy.replayweb.page/embed/\\1"
         },
         {
-            "pattern": ".*(?:gcs-vimeo|vod|vod-progressive)\\.akamaized\\.net.*?/([\\d/]+.mp4)$",
-            "replace": "vimeo-cdn.fuzzy.replayweb.page/\\1"
+            "pattern": ".*(?:gcs-vimeo|vod|vod-progressive|vod-adaptive)\\.akamaized\\.net.*/(.+?.mp4)\\?.*range=(.*?)(?:&|$)",
+            "replace": "vimeo-cdn.fuzzy.replayweb.page/\\1?range=\\2"
```

Comment:
Where this change come from ?

You find it or it is coming from wabac update ?

Reply:
I found it. To be reported upstream as well.

Reply:
Done: https://github.com/webrecorder/wabac.js/issues/169